src/cli.ts

```diff
@@ -14,19 +14,18 @@ program
   .version(version, '-v, --version')
   .option('-i, --input <file>', 'File path (or URL) to OpenAPI Specification, i.e. swagger.json (required)')
   .option('-o, --output <directory>', 'Path to output directory for generated TypeScript code (required)')
+  .option('-f, --force', 'Force deleting the output directory before generating')
   .parse(process.argv);
 
 if (!program.input || !program.output) {
   program.outputHelp();
   process.exit(1);
 }
 
-const outputDirectory = path.resolve(process.cwd(), program.output);
+const outputDirectory = path.resolve(program.output || '.');
```

Comment:
`path.resolve(process.cwd())` and `path.resolve('.')` result in the same directory.